src/components/tabs/tabs.njk

```diff
@@ -1,35 +1,35 @@
-<div class="rvt-tabs" data-tabset="tabset-1">
+<div class="rvt-tabs" data-rvt-tabs="tabset-1">
     <div class="rvt-tabs__tablist" role="tablist" aria-label="Rivet tabs">
         <button class="rvt-tabs__tab"
                 role="tab"
                 aria-selected="false"
-                data-tab="tab-1"
+                data-rvt-tab="tab-1"
                 id="t-one">
             Tab one
         </button>
         <button class="rvt-tabs__tab"
                 role="tab"
                 aria-selected="false"
-                data-tab="tab-2"
+                data-rvt-tab="tab-2"
                 id="t-two">
             Tab two
         </button>
         <button class="rvt-tabs__tab"
                 role="tab"
                 aria-selected="false"
-                data-tab="tab-3"
+                data-rvt-tab="tab-3"
                 id="t-three">
             Tab three
         </button>
         <button class="rvt-tabs__tab"
                 role="tab"
                 aria-selected="false"
-                data-tab="tab-4"
+                data-rvt-tab="tab-4"
                 id="t-four">
             Tab four
         </button>
     </div>
-    <div class="rvt-tabs__panel" tabindex="0" role="tabpanel" id="tab-1" aria-labelledby="t-one" data-tab-panel="tab-1" hidden>
+    <div class="rvt-tabs__panel" tabindex="0" role="tabpanel" id="tab-1" aria-labelledby="t-one" data-rvt-tab-panel="tab-1" hidden>
```

Comment:
Similar story here. Following some progressive enhancement principles might be the way to go, where we, the component authors, add the hidden attributes in our set up code. that way if something goes wrong on the client or in someone's app, the content is still accessible even though the toggle/tabs controls might not be working.